tests/parsers/test_android.py

```diff
@@ -291,6 +291,37 @@ def test_get_and_set_str(derived):
 
     np.testing.assert_equal(derived[key, :], value)
 
+def test_android_concat(derived, pd_df):
+    """Test concat on Android data.
+
+    Parameters
+    ----------
+    derived : AndroidDerived2021
+        Instance of AndroidDerived2021 for testing
+    pd_df : pytest.fixture
+        pd.DataFrame for testing measurements
+    """
+
+    # remove first timestamp to match
+    pd_df = pd_df[pd_df['millisSinceGpsEpoch'] != pd_df.loc[0,'millisSinceGpsEpoch']]
+
+    # extract and combine gps and glonass data
+    gps_data = derived.where("gnss_id","gps")
+    glonass_data = derived.where("gnss_id","glonass")
+    combined_navdata = gps_data.concat(glonass_data)
```

Comment:
Right. Order does matter. A concatenated to B is different than B concatenated to A

Reply:
Added comparison of alternate of glonass + gps as suggested. That ordering difference was already tested in `test_concat` but doesn't hurt to add another for assurance since that test doesn't take long.